sample-dapps/ethereum-wallet-explorer/src/components/explorer/nfts/NFTsTable.tsx

```diff
@@ -0,0 +1,105 @@
+// NFTsTable.tsx
+"use client"
+import React, { useEffect, useState } from 'react';
+import { TableCell, TableRow, TableBody, Table, TableHeader, TableHead } from "../../ui/table";
+import { Button } from '../../ui/button';
+import TableSkeleton from '../helpers/TableSkeleton';
+import { json } from 'stream/consumers';
+
+interface NFT {
+    collectionName: string;
+    collectionTokenId: string;
+    collectionAddress: string;
+    name: string;
+    description: string;
+    imageUrl: string;
+    chain: string;
+    network: string;
+}
+
+interface NFTProps {
+    walletAddress: string;
+}
+
+const NFTsTable = ({ walletAddress }: NFTProps) => {
+    const [nfts, setNfts] = useState<NFT[]>([]);
+    const [loading, setLoading] = useState(true);
```

Comment:
do you want default to be true? i think false. then it'll start when your useEffect triggers. If for some reason the useEffect doesn't trigger, the loader would just keep going even if nothing is happening.